pkgs/development/libraries/ffmpeg/generic.nix

```diff
@@ -671,7 +677,8 @@ stdenv.mkDerivation rec {
       ++ optional withGPL gpl2Plus
       ++ optional withGPLv3 gpl3Plus
       ++ optional withUnfree unfreeRedistributable;
+    pkgConfigModules = [ "libavutil" ];
```

Comment:
pkg-config? [no problem!](https://www.youtube.com/watch?v=9kaIXkImCAM)

Also `libavutil` - what? - and ok then.